### DISCUSSION TITLE
When the connect wallet modal pops up why does a right-padding of 17px get applied to the body element?

### DISCUSSION THREAD

**Discussion by sjunaiduk** [family/connectkit#185] [Q&A] [answered]
Hey, when I click the connect wallet button, on the opening of the modal, the body tag of the document gets the following styles applied inline automatically:

`position: relative;
overflow: hidden;
touch-action: noe;
padding-right: 17px`

When the modal opens the entire page shifts to the left by 17px which doesn't look good, I can override this by just setting the padding-right of the body to 0 and marking it as important, but it isn't a clean solution and I don't know why I should be doing this in the first place.

Is there anyway to disable this and why is this happening?

**Comment by lochie** [comment, marked as answer]
Hey @sjunaiduk, this is by design for few reasons, but you can disable this feature by using [avoidLayoutShift](https://docs.family.co/connectkit/api-reference#avoidLayoutShift)
